tests/std/tests/P0645R10_text_formatting_utf8/test.cpp

```diff
@@ -82,8 +82,8 @@ void test_width_estimation() {
         {TYPED_LITERAL(CharT, "\u2e80\x58"), 3},
         {TYPED_LITERAL(CharT, "\u303e\x58"), 3},
         {TYPED_LITERAL(CharT, "\u303f\x58"), 2},
-        {TYPED_LITERAL(CharT, "\u3040\x58"), 3},

```

Comment:
The comments have also been outdated for these cases
```
// test the boundaries of the intervals defined in n4885 [format.string.std]/11
// skip over the surrogate pair range (\ud800-\udfff)
```
Is it better to just comment-out these test cases?
// {TYPED_LITERAL(CharT, "\u3040\x58"), 3}, becomes invalid after adopting ...